qark/modules/filePermissions.py

```diff
@@ -28,9 +28,9 @@ def start(queue, height):
     file_wr = r'MODE_WORLD_READABLE'
     for i in text_scan(common.java_files, file_wr):
         if len(i) > 0:
-            report.write(IssueType.FileSystem, IssueSeverity.High,
-                         common.config.get('qarkhelper', 'WR_FILE') + str(i[0])
-                         + "<br>" + str(i[1]))
+            #report.write(IssueType.FileSystem, IssueSeverity.High,
+            #             common.config.get('qarkhelper', 'WR_FILE') + str(i[0])
+            #             + "<br>" + str(i[1]))
```

Comment:
Commented this out, it is redundant with lines 34-44 below and also throws errors when compiling the report.